internal/testutil/mig/test_case.go

```diff
@@ -4,16 +4,19 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 	"github.com/stretchr/testify/require"
 )
 
 func CreateAndRunTest(t *testing.T, test *resource.TestCase) {
 	t.Helper()
+	acc.SkipInUnitTest(t)
```

Comment:
What's the reason for this? Do we want this for all mig tests?
From the method definition I see:
`SkipInUnitTest allows skipping a test entirely when TF_ACC=1 is not defined. This can be useful for acceptance tests that define logic prior to resource.Test/resource.ParallelTest functions as this code would always be run.`

Reply:
I think so since they will create resources in the cloud. 
All migration tests need the `MONGODB_ATLAS_LAST_VERSION` (see [Here for a failed unit test](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/8450359863/job/23146358963)).
Therefore, I think this is in line with our current practices since this env-var is NOT set in our unit test pipeline.

Reply:
is it worth it adding a comment in the file itself? it might be confusing and lead the developer going through files to understand why

Reply:
Added comment in [8d5f7d0](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2094/commits/8d5f7d0db20e9ff2f2824df020691417a5f351e5)

Reply:
we want to use SkipInUnitTest when it's not detected automatically for acc or mig tests.
For example resource.Test/ParallelTest handle that, but it's needed before calling them, e.g. when declaring vars in the test, or here in this mig function that is called before  resource.Test/ParallelTest